sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/JacksonAdapter.java

```diff
@@ -153,8 +172,8 @@ public String serializeList(List<?> list, CollectionFormat format) {
 
     @Override
     @SuppressWarnings("unchecked")
-    public <T> T deserialize(String value, final Type type, SerializerEncoding encoding) throws IOException {
-        if (CoreUtils.isNullOrEmpty(value)) {
+    public <T> T deserialize(byte[] value, final Type type, SerializerEncoding encoding) throws IOException {
```

Comment:
This is an API breaking change.

Reply:
It's now called deserializeFromBytes in more recent commits.